### ISSUE TITLE
[INFO] Your Pokemon inventory is full! Could not catch!

### ISSUE THREAD

**Issue by malaypatel11** [PokemonGoF/PokemonGo-Bot#5485]
### Expected Behavior

Pokemon not caching issue occur after upgrading new version. 

### Actual Behavior

[2016-09-16 21:45:48] [MainThread] [PokemonCatchWorker] [INFO] Your Pokemon inventory is full! Could not catch!

### Your FULL config.json (remove your username, password, gmapkey and any other private info)
http://pastebin.com/Exj4Fe0z

### Output when issue occurred

[2016-09-16 21:45:45] [MainThread] [PokemonCatchWorker] [INFO] Your Pokemon inventory is full! Could not catch!
[2016-09-16 21:45:46] [MainThread] [PokemonCatchWorker] [INFO] Your Pokemon inventory is full! Could not catch!
[2016-09-16 21:45:48] [MainThread] [PokemonCatchWorker] [INFO] Your Pokemon inventory is full! Could not catch!

### Steps to Reproduce
Also change config.json to evolve pokemon but i am getting error. can you please help me in config.json file to resolve ?

### Other Information
OS: 
OSX

Branch: 
dev

Git Commit: 
5ae0afc9e770cc3b6d6bdc31170fa080ed4a259b

Python Version: 
Python 2.7.10

**Comment by mmixx** [comment]
To fix this... you just need to add on your config the filter `"release_below_ivcp": 0.5`... you can put any value...

```
    "release": {
      "Caterpie": { "release_below_cp": 656, "release_below_iv": 0.8, "release_below_ivcp": 0.5, "logic": "and" },
      "Weedle": { "release_below_cp": 656, "release_below_iv": 0.8, "release_below_ivcp": 0.5, "logic": "and" },
      "Diglett": { "release_below_cp": 658, "release_below_iv": 0.8, "release_below_ivcp": 0.5, "logic": "and" },
      "Metapod": { "release_below_cp": 668, "release_below_iv": 0.8, "release_below_ivcp": 0.5, "logic": "and" }
    },
```

**Comment by sungsel** [comment]
or u can try this script.
```
    "release": {
      "Caterpie": {"keep_best_custom": "iv, cp, hp_max", "amount":3},
      "Zubat": {"keep_best_custom": "iv, cp, hp_max", "amount":3},
      "Weedle": {"keep_best_custom": "iv, cp, hp_max", "amount":3},
      "Diglett": {"keep_best_custom": "iv, cp, hp_max", "amount":3},
    },

```

**Comment by Gobberwart** [comment]
Likely fixed by #5499 
Please update and confirm/close this.

**Comment by malaypatel11** [comment]
Thanks now working ...
